glide.lock

```diff
@@ -1,28 +1,28 @@
-hash: 5fc263bfacc703e0a599166ccae968891dd53d7b4e42a3d8356b3194e45833bb
-updated: 2018-06-01T15:20:52.053211272+02:00
+hash: a7e0c0723a91776f34ff8a742912c032e115701284efe18eed15e96da0ed00fd
```

Comment:
This is the hash of the `glide.yaml` file.

Reply:
I wasn't aware that the hash only depends on `glide.yaml`. Then it's easy of course. I pushed a new version, that only changes the hash and the negroni version in `glide.lock`.